charts/wg-access-server/README.md

```diff
@@ -18,48 +18,52 @@ If no admin password is set, the Chart generates a random one. You can retrieve
 
 ## Uninstalling the Chart
 
-To uninstall/delete the my-release deployment:
+To uninstall/delete the `wireguard` deployment:
 
-```bash
-$ helm delete my-release
+```
+$ helm delete wireguard
 ```
 
 The command removes all the Kubernetes components associated with the chart and deletes the release.
 
 ## Example values.yaml
 
-```yaml
-config:
-  externalHost: "<loadbalancer-ip>"
-
-# wg access server is an http server without TLS. Exposing it via a loadbalancer is NOT secure!
-# Uncomment the following section only if you are running on private network or simple testing.
-# A much better option would be TLS terminating ingress controller or reverse-proxy.
-# web:
-#   service:
-#     type: "LoadBalancer"
-#     loadBalancerIP: "<loadbalancer-ip>"
+```
+# wg-access-server config
+web:
+  config:
+    adminUsername: "<Username for the admin user>"
+    adminPassword: "<Password for the admin user>",
+  service:
+    type: 'LoadBalancer',
+    loadBalancerIP: "IP of the admin panel",
 
 wireguard:
   config:
-    privateKey: "<wireguard-private-key>"
+    privateKey: "<Private Key>"
   service:
-    type: "LoadBalancer"
-    loadBalancerIP: "<loadbalancer-ip>"
+    type: ClusterIP
+    loadBalancerIP: "IP of the WireGuard service"
 
 persistence:
   enabled: true
+  size: "100Mi"
+  accessModes:
+    - ReadWriteOnce
 
 ingress:
   enabled: true
-  hosts: ["vpn.example.com"]
+  annotations:
+    kubernetes.io/ingress.class: "nginx",
+    cert-manager.io/cluster-issuer: "letsencrypt-prod" 
+  hosts:
+    - vpn.mydomain.com
   tls:
-    - hosts: ["vpn.example.com"]
-      secretName: "tls-wg-access-server"
+    - hosts: 
+        - vpn.mydomain.com
```

Comment:
The same as the other occurrence of mydomain.com

Reply:
make sense, changed!

Reply:
@catrielmuller Thanks for the changes. However, with the suggestion to change I meant only the TLD and not the subdomain. The idea to use the subdomain `vpn` was good. I would change it to avoid confusion about the domain later. Otherwise the PR looks good so far. Now we only have to wait for some of the other reviewers to report back whether this can be merged.